permissions/plugin-caffeine-api.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: "caffeine-api"
+github: &GH "jenkinsci/caffeine-api-plugin"
```

Comment:
@daniel-beck is this some new funky syntax or a bug in the codethat created the PR?

Reply:
it's a yaml anchor and is used if you're referring to the github repo later in the file, see https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/plugin-pull-request-monitoring.yml#L9

Reply:
Unnecessary when the issue tracker is Jira, so strictly speaking it's a tiny little bug 🤷